packages/@coorpacademy-components/src/molecule/cm-popin/index.js

```diff
@@ -33,12 +40,57 @@ const CMPopin = props => {
         backgroundSize: 'cover'
       }
     : null;
-
+  const renderHeader = () => {
+    if (header) return <img className={style.headerBackground} src={header} />;
+    if (mode === 'cookie')
+      return (
+        <div className={style.cookieHeader}>
+          <div className={style.cookieIconContainer}>
+            <Cookie className={style.cookieIcon} />
+          </div>
+          <div className={style.cookieTitle}>{cookieTitle}</div>
+        </div>
+      );
+    return null;
+  };
+  const getClassBtnSwitch = (index, btnList) => {
+    switch (index) {
+      case 0:
+        return style.firstBtnSwitchContainer;
+      case btnList.length - 1:
+        return style.lastBtnSwitchContainer;
+      default:
+        return style.singleSwitchContainer;
+    }
+  };
+  const renderBtnSwitch = () => {
+    return map.convert({cap: false})((el, index) => (
+      <div key={index} className={getClassBtnSwitch(index, listBtnSwicth)}>
+        <InputSwitch
+          {...{
+            type: el.type,
```

Comment:
après destructuration ce sera plus concis
```suggestion
            type,
            title,
            ...
```